test/e2e.v3.test.ts

```diff
@@ -171,15 +171,6 @@ describe("v3 E2E Test Scenarios", () => {
       expect(verified).toBe(true);
       expect(validateSchema(obfuscatedDocument)).toBe(true);
     });
-    test("obfuscate data transistively", async () => {
```

Comment:
why removing this ?

Reply:
ohh because I shifted the test to obfuscate but I can put it back!